.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 .AppleDouble
 .LSOverride
+*.xcodeproj
```

Comment:
Why this change? Looks like the project file is in the repo, which is extremely important (as explained in #117).